crates/rpc/eth/block.rs

```diff
@@ -1,4 +1,6 @@
-use ethereum_rust_chain::find_parent_header;
```

Comment:
To make this more similar to how the apis are divided in https://github.com/ethereum/execution-apis/blob/main/src/eth/fee_market.yaml, I would move this to /eth/fee_market.rs

Reply:
Oh sure! I didn't know about that resource.